velocity-transition-group.js

```diff
@@ -44,6 +44,13 @@ var ReactDOM = require('react-dom');
 var ReactTransitionGroup = require('react-addons-transition-group');
 var Velocity = require('./lib/velocity-animate-shim');
 
+// Shim requestAnimationFrame for browsers that don't support it, in particular IE 9.
+var shimRequestAnimationFrame = 
```

Comment:
Yeah, I think that's fair.

On Fri, Jan 15, 2016 at 4:57 PM, Rand Scullard <notifications@github.com>
wrote:

> In velocity-transition-group.js
> <https://github.com/twitter-fabric/velocity-react/pull/65#discussion_r49910027>
> :
>
> > @@ -44,6 +44,13 @@ var ReactDOM = require('react-dom');
> >  var ReactTransitionGroup = require('react-addons-transition-group');
> >  var Velocity = require('./lib/velocity-animate-shim');
> >
> > +// Shim requestAnimationFrame for browsers that don't support it, in particular IE 9.
> > +var shimRequestAnimationFrame =
>
> Sorry, I didn't think of the server case. So shimRequestAnimationFrame can
> just be set to null if window is undefined?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twitter-fabric/velocity-react/pull/65/files#r49910027>
> .
>